### DISCUSSION TITLE
Pseudo-static analysis in Anura 3D 2022

### DISCUSSION THREAD

**Discussion by MiguelSC98** [Anura3D/Anura3D_OpenSource#58] [General]
Dear all.
Hi.
I want to know if it's possible to develop pseudostatic slope analysis in Anura 3D 2022 (pre processor GID 14.0.1) and what steps I must follow.
Thank you in advance for your help.

**Comment by murphy-john** [comment]
I believe this can be done using the "Quasi-static" calculation method discussed in the Tutorial Manual.

**Comment by ayerrocolom** [comment]
I believe that you should be able to do that by changing the orientation of the gravity vector and its magnitude in the CPS file accordingly.
